src/tests/profiler/native/releaseondetach/releaseondetach.cpp

```diff
@@ -51,6 +53,17 @@ ReleaseOnDetach::~ReleaseOnDetach()
 
     fflush(stdout);
 
+
+    for (int i = 0; i < 50000; ++i)
+    {
+        if  (_doneFlag != NULL)
+        {
+            break;
+        }
+
+        std::this_thread::sleep_for(std::chrono::milliseconds(1));
```

Comment:
These types of timeouts tend to be fragile.  Is this long enough for gcstress runs?

Reply:
It has been running with a timeout for a while and this is the first failure of this type I've seen so I was trying to be lazy. I should just fix it the right way though, I just pushed a commit getting rid of all the sleeps.